spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
Ideally the factories don't need to know the meaning of hardcoded ID values in the database in order to function correctly. Doing it this way means that this factory now has an implicit dependency on specific data being in the `Option` and `OptionSet` tables and they're not referenced in the factory at all. 

There's a few different ways we can handle this, but they all involve making sure that before these traits are used the loan factory has a valid reference to an `OptionSet`. You can see [the changes I made to the loan factories for translations](/madeline_system/blob/3760_loans_index/spec/factories/loans.rb#L67) as a starting point, but it would probably be helpful to look over some other stuff in #19 in general to see how these option sets are being used both in the specs and in the controller and see if there's a way to closely replicate the interface they're using since we'll need it for more complex filtering later.